draft-ietf-gnap-core-protocol.md

```diff
@@ -653,24 +704,87 @@ expired access token at the AS using the token's management URL.
     [access token](#response-token) usable at the RS. The access token
     response includes a token management URI.
     
-3. The client instance [presents the token](#use-access-token) to the RS. The RS 
-    validates the token and returns an appropriate response for the
+3. The client instance [uses the access token](#use-access-token) to call the RS.
+
+4. The RS validates the access token and returns an appropriate response for the
     API.
-    
-4. When the access token is expired, the RS responds to the client instance with
-    an error.
-    
-5. The client instance calls the token management URI returned in (2) to
+ 
+5. Time passes and the client instance uses the access token to call the RS again.
+   
+6. The RS validates the access token and determines that the access token is expired
+    The RS responds to the client instance with an error.
+
+7. The client instance calls the token management URI returned in (2) to
     [rotate the access token](#rotate-access-token). The client instance
-    presents the access token as well as the appropriate key.
+    [uses the access token](#use-access-token) in this call as well as the appropriate key,
+    see the token rotation section for details.
 
-6. The AS validates the rotation request including the signature
+8. The AS validates the rotation request including the signature
     and keys presented in (5) and returns a 
     [new access token](#response-token-single). The response includes
     a new access token and can also include updated token management 
     information, which the client instance will store in place of the values 
     returned in (2).
-   
+
+### Requesting User Information {#sequence-user}
+
+In this scenario, the client instance is not calling an RS, and does not
```

Comment:
Will
`In this scenario, the client instance neither calls an RS, nor requests an access token.`
or
`In this scenario, the client instance is neither calling an RS, nor requesting an access token.`
or 
`In this scenario, the client instance does not call an RS and does not request an access token. `
work better?

Reply:
You're right, that is better. Thanks!